src/test-utils/test-utils.ts

```diff
@@ -58,20 +58,30 @@ export const PUT_RUM_EVENTS_REQUEST: PutRumEventsRequest = {
     ]
 };
 
+export const WEB_CLIENT_VERSION = '2.0.0';
```

Comment:
question: This is unused. Are you planning on adding the version info in a future PR?

Reply:
Refactored